Header header
uint8 marker_count
Marker[] markers
